src/other-reprs.md

```diff
@@ -15,18 +15,26 @@ or C++. Any type you expect to pass through an FFI boundary should have
 necessary to soundly do more elaborate tricks with data layout such as
 reinterpreting values as a different type.
 
-However, the interaction with Rust's more exotic data layout features must be
+We strongly recommend using [rust-bindgen][] and/or [cbdingen][] to manage your FFI
```

Comment:
markdown ProTip: you don't need the empty `[]`s

Reply:
You can leave them out, but using them makes the content slightly more robust and unambiguous, similar to quoting keys in JS